src/page-response-impact/ReducingR0ImpactMetrics.tsx

```diff
@@ -36,15 +36,15 @@ const ReducingR0ImpactMetrics: React.FC<Props> = ({ populationImpact }) => {
       <ImpactMetricCard
         title={
           <HospitalBedsTitle
-            title="Hospital beds used at peak"
+            title="Hospital beds used by incarcerated"
             value={populationImpact.hospitalBedsUsed}
           />
         }
         value={`${formatAbsValue(populationImpact.hospitalBedsUsed)}%`}
         icon={hospitalIcon}
       />
       <ImpactMetricCard
-        title={`Additional staff ${staffAbilityText} to work at peak`}
+        title={`Additional staff ${staffAbilityText} to work`}
```

Comment:
Are these the latest and greatest copy suggestions? @rocktacular merged a PR this morning adding the 'at peak' copy. Is this the final?

Reply:
yeah after chatting with @sychang we were trying to find a way to shorten the card text to include `incarcerated` and she had an idea of moving `peak` to the heading instead. I'm still waiting for Serena to confirm these screenshots